db/migrate/20200323191741_create_user_collections.rb

```diff
@@ -3,7 +3,9 @@ def change
     create_table :user_collections do |t|
       t.string :title
       t.references :user, foreign_key: true
-      t.timestamps 
+      t.timestamps
+
+      add_index(:user_collections, :user)
```

Comment:
Glad we have timestamps here. I think it's always a good idea have this in place to keep track.